test/integration/api-schema/download-json-schemas.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -x
+
+schemadir="${1:-.schemacache}"
+
+FILES=( \
+  "errors/error.json" \
+  "errors/payload.json" \
+  "sourcemaps/payload.json" \
+  "transactions/mark.json" \
+  "transactions/payload.json" \
+  "transactions/span.json" \
+  "transactions/transaction.json" \
+  "context.json" \
+  "process.json" \
+  "request.json" \
+  "service.json" \
+  "stacktrace_frame.json" \
+  "system.json" \
+  "user.json" \
+)
+
+mkdir -p ${schemadir}/errors ${schemadir}/transactions ${schemadir}/sourcemaps
+
+for i in "${FILES[@]}"; do
+  output="${schemadir}/${i}"
+  curl -sf --compressed https://raw.githubusercontent.com/elastic/apm-server/master/docs/spec/${i} > ${output} || exit $?
```

Comment:
Would it make sense to download from the `6.x` or the `6.2` branch instead? Otherwise we might suddenly get breaking tests if the schema is intentionally updated in a breaking way. The downside of that is that we'll not get notified on those updates, so we'll have to remember to update it - or have a separate test build that pulls from master and allows for failures.

We could also just keep it as is for now and update the process once it becomes an issue